src/server/render/default-document/index.js

```diff
@@ -22,9 +21,7 @@ const escapeScriptTags = data => {
 const getProductionBundle = () => {
   const assetsPath = path.resolve(process.cwd(), '.tapestry', 'assets.json')
   const assets = fs.readJsonSync(assetsPath)
-  return map(assets, asset => (
-    <script src={asset.js} />
-  ))
+  return Object.values(assets).map(({ js })=> (<script src={js} />))
```

Comment:
that was totally pseudo code. _it works??_

Reply:
yep just added implicit JSX return 👌